Sources/muterCore/MutationOperators/TokenAwareVisitor.swift

```diff
@@ -4,7 +4,9 @@ class TokenAwareVisitor: SyntaxVisitor, PositionDiscoveringVisitor {
     
     fileprivate(set) var tokensToDiscover = [TokenKind]()
     private(set) var positionsOfToken = [AbsolutePosition]()
-    
+
+    init(_: MuterConfiguration) { }
```

Comment:
Why in the RemoveSideEffectsOperator you are exposing the parameter's name and here you are not?  I can see that you are not using it in the TokenAwareVisitor implementation, but it would be nice to follow the same style for both structures. What do you think?

Reply:
(This may depend on the resolution of the next comment)
Yes, I omitted the parameter name here since it's not used, but I see your point. In particular, if the initialiser were to be declared as a protocol requirement at some point, the parameter name will have to match, if I'm not mistaken, so I'll add it.